src/stores/WorkflowTypes.ts

```diff
@@ -0,0 +1,53 @@
+import { Constraint } from "./ConstraintStore";
+import { Domain } from "./DomainStore";
+
+// Currently unused
+export type Workflow = {
+  id: string;
+  label: string;
+  domain: Domain;
+  lengths: Number[]; //TODO: what type is this?
```

Comment:
Does this describe the length of the workflow or something else?

Reply:
I don't remember exactly what I originally based it on, but I think this is now superseded by WorkflowSolution and this one can be removed.